modules/tensor_mechanics/doc/content/source/materials/CZM/CZM3DCLaw.md

```diff
@@ -0,0 +1,36 @@
+# CZM 3DC traction separation law
+
+!syntax description /Materials/CZM3DCLaw
+
+##Description
+
+This class implements the non-stateful traction separation law proposed by [!cite](salehani2018coupled). This traction separation law is optimal for monotonic loading conditions because damage is not incorporated. This model can be used for both 1D, 2D and 3D problems.
```

Comment:
change to:
This traction separation law should only be used for monotonic loading conditions as it will not produce realistc results for unloading and reloading.